.github/PULL_REQUEST_TEMPLATE/port.md

```diff
@@ -0,0 +1,6 @@
+Port <FROM_BRANCH> to <TO_BRANCH>
```

Comment:
I think we can remove this file now

Reply:
Thanks for catching 144196c84551cab4bb3a5bf0e9ea68a39b9c944d